Python/pylifecycle.c

```diff
@@ -687,6 +687,10 @@ pycore_init_global_objects(PyInterpreterState *interp)
 
     _PyUnicode_InitState(interp);
 
+    if (_Py_IsMainInterpreter(interp)) {
+        _Py_GetConstant_Init();
```

Comment:
I'm thinking ahead a little bit here, and we could change this later, but I think I'd pass `interp` into the init function (and it can do the `IsMain` check now, or later change to some other interpreter-specific behaviour if it becomes interesting/necessary).

@ericsnowcurrently FYI.